config/owasp/suppressions.xml

```diff
@@ -10,4 +10,31 @@
     <cve>CVE-2014-0054</cve>
     <cve>CVE-2016-1000027</cve>
   </suppress>
+  <suppress until="2022-05-01">
+    <notes><![CDATA[
+   file name: spring-plugin-core-2.0.0.RELEASE.jar
+   ]]></notes>
```

Comment:
I don't think these are needed anymore as I pushed in a suppression PR and removed alot of these.